plugins/webhooks/index.js

```diff
@@ -253,7 +253,11 @@ async function triggeredPipelines(pipelineFactory, scmConfig, branch, type, acti
     let pipelinesOnOtherBranch = [];
 
     pipelines.forEach((p) => {
-        if (p.scmUri.startsWith(scmUri)) {
+        // This uri expects 'scmUriDomain:repoId:branchName:rootDir'. To Compare, rootDir is ignored.
+        const tmpScmUri = p.scmUri.split(':');
+        const tmpScmBranch = `${tmpScmUri[0]}:${tmpScmUri[1]}:${tmpScmUri[2]}`;
```

Comment:
I think we should rename this parameter for more information.

Reply:
Thank you. I renamed.